index.html

```diff
@@ -4,13 +4,181 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./css/style.css" />
-    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="./css/index.css" />
+    <title>MY PORTFOLIO</title>
+    <link rel="stylesheet" href="index.css" />
   </head>
-
-  <body>
-    <header>
-      
+  <body class="position-relative body">
+    <header class="display-flex nav-bar justify-content-space-between">
+      <div class="name-div">
+        <a href="#"> Basit Korede </a>
+      </div>
+      <nav>
+        <button type="button" class="toggle-button">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
+      </nav>
     </header>
+    <main class="main-page">
+      <svg
+        class="ellipse-18 position-absolute"
+        viewBox="0 0 104 260"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g filter="url(#filter0_f)">
+          <circle cx="-26" cy="130" r="80" fill="#EDECFF" />
+        </g>
+        <defs>
+          <filter
+            id="filter0_f"
+            x="-156"
+            y="0"
+            width="260"
+            height="260"
+            filterUnits="userSpaceOnUse"
+            color-interpolation-filters="sRGB"
+          >
+            <feFlood flood-opacity="0" result="BackgroundImageFix" />
+            <feBlend
+              mode="normal"
+              in="SourceGraphic"
+              in2="BackgroundImageFix"
+              result="shape"
+            />
+            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur" />
+          </filter>
+        </defs>
+      </svg>
+      <svg
+        class="ellipse-19 position-absolute"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g filter="url(#filter0_f)">
+          <circle cx="84" cy="84" r="34" fill="#EDECFF" />
+        </g>
+        <defs>
+          <filter
```

Comment:
Please ensure that you must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![your_design](https://user-images.githubusercontent.com/10332499/153178565-466a4161-5dc2-4d2b-9180-29eccda6e59d.PNG) | ![figma_design](https://user-images.githubusercontent.com/10332499/153178605-14725a04-6f98-4022-8e99-85b0c6b2683f.PNG)

Reply:
Hello sir I have made changes to this before creating a pull request please help me check well. Thanks

Reply:
@BasitKorede , you can call me Michael. Kindly re-submit for review on your dashboard.

Reply:
Thanks, @micheaol for the review and catching the bug. 👋 

It seems I broke the page by linking the wrong CSS file, [it is fixed now and should work as expected](https://raw.githack.com/BasitKorede/my-professional-portfolio/add-setup-moble-version-skeleton/index.html).

I will resubmit for another review. Thanks again! 🙂